### DISCUSSION TITLE
I got fatal error when run the example code

### DISCUSSION THREAD

**Discussion by albertxu1** [halaxa/json-machine#82] [General]
I am new to PHP, but I got fatal error when run the example code. 

> Fatal error: Uncaught Error: Class "JsonMachine\Items" not found in /opt/lampp/htdocs/test.php:9 Stack trace: #0 {main} thrown in /opt/lampp/htdocs/test.php on line 9

I installed the json-machine with composer, the version is 1.1
```
{
    "require": {
        "loilo/fuse": "^6.6",
        "halaxa/json-machine": "^1.1"
    }
}
```

```
<?php

use \JsonMachine\Items;

// this often causes Allowed Memory Size Exhausted
// $users = json_decode(file_get_contents('500MB-users.json'));

// this usually takes few kB of memory no matter the file size
$users = Items::fromFile('services.json');

foreach ($users as $id => $user) {
    // just process $user as usual
    var_dump($user->name);
}
```

**Comment by halaxa** [comment]
Hi, make sure to run `composer update` after editing composer.json and double check you correctly init composer's autoloading. If you look at the tests of JSON Machine you'll see that `JsonMachine\Items` is used all over the place and works correctly. Let me know how it goes.

**Reply by halaxa** [reply]
Also maybe composer autoloader php file is missing...
`require_once 'vendor/autoload.php';`
